warehouse/internal/repo/schema_test.go

```diff
@@ -142,5 +142,50 @@ func TestWHSchemasRepo(t *testing.T) {
 		t.Log("empty")
 		_, err = r.GetTablesForConnection(ctx, []warehouseutils.SourceIDDestinationID{})
 		require.EqualError(t, err, errors.New("no source id and destination id pairs provided").Error())
+
+		t.Log("multiple")
+		latestNamespace := "latest_namespace"
+		sourceID2 := "source_id_2"
+		destinationID2 := "destination_id_2"
+		connection2 := warehouseutils.SourceIDDestinationID{SourceID: sourceID2, DestinationID: destinationID2}
+		schemaLatest := model.WHSchema{
+			UploadID:        2,
+			SourceID:        sourceID,
+			Namespace:       latestNamespace,
+			DestinationID:   destinationID,
```

Comment:
Shouldn't this be `destinationID2`? :thinking: 
Same with the `sourceID` above. I thought you would insert two records with the same source and destination and then expect only one in `GetTablesForConnection`.